css/bootstrap-slider.css

```diff
@@ -243,4 +244,75 @@
   background-repeat: repeat-x;
   filter: progid:DXImageTransform.Microsoft.gradient(startColorstr='#ff89cdef', endColorstr='#ff81bfde', GradientType=0);
   opacity: 1;
-}
+  background-color:#89cdef;
```

Comment:
can't remember that i have added the background-color maybe it was just for some test.